utils/check_repo.py

```diff
@@ -273,9 +273,46 @@ def check_all_models_are_documented():
         raise Exception(f"There were {len(failures)} failures:\n" + "\n".join(failures))
 
 
+_re_decorator = re.compile(r"^\s*@(\S+)\s+$")
+
+
+def check_decorator_order(filename):
+    """ Check that in the test file `filename` the slow decorator is always last."""
+    with open(filename, "r", encoding="utf-8") as f:
+        lines = f.readlines()
+    decorator_before = None
+    errors = []
+    for i, line in enumerate(lines):
+        search = _re_decorator.search(line)
+        if search is not None:
+            decorator_name = search.groups()[0]
+            if decorator_before is not None and decorator_name.startswith("parametrized"):
```

Comment:
I don't seem to have perms to make suggestions, but it should be `parameterized` - i.e. s/tr/ter/ (missing `e`)

it's different for pytest where it's `parametrize` w/o `e`

Reply:
Misread your comment, sorry about that.

Reply:
it's easy to miss - I think they had to rename it slightly to avoid collision with pytest's decorator, leading to even more confusion.